src/fsharp/TypeChecker.fs

```diff
@@ -1405,31 +1405,32 @@ let ComputeAccessAndCompPath env declKindOpt m vis overrideVis actualParent =
     let cpath = if accessModPermitted then Some env.eCompPath else None
     vis, cpath 
 
-let CheckForAbnormalOperatorNames cenv (idRange:range) opName isMember =    
+let CheckForAbnormalOperatorNames cenv (idRange:range) coreDisplayName memberInfoOpt =
     if (idRange.EndColumn - idRange.StartColumn <= 5) && 
         not cenv.g.compilingFslib 
     then
+        let opName = DecompileOpName coreDisplayName
```

Comment:
Not every branch of the pattern uses DecompileOpName.
With this change, it will happen regardless of the branch executed.  Im guessing that it will now be called more times.  
I suppose you can make it lazy with: let opName () = DecompileOpName coreDisplayName

Kevin

Reply:
Okay upon closer look, this looks good.  Ignore my prvious comment.